client/my-sites/customer-home/main.jsx

```diff
@@ -152,7 +152,7 @@ class Home extends Component {
 				) }
 				{ siteId && ! hasChecklistData && <QuerySiteChecklist siteId={ siteId } /> }
 				{ hasChecklistData &&
-					( isChecklistComplete ? (
+					( isAtomic || isChecklistComplete ? (
```

Comment:
I like this way of rendering better!

I still think a comment here about why the checklist is hidden for Atomic sites is a good idea, probably with a link to pb5gDS-7c-p2